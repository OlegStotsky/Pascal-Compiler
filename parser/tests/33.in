procedure a (a : integer)
begin
a := 5;
end