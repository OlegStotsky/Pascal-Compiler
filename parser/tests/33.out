BEGIN GLOBAL SYM TABLE
 BEGIN GLOBAL SYM TABLE VARS
  PROCEDURE a
   BEGIN LOCAL SYM TABLE
    BEGIN LOCAL SYM TABLE VARS
     VAR a:integer
    END LOCAL SYM TABLE VARS
   END LOCAL SYM TABLE
   BEGIN STMT BLOCK
    BEGIN STMT ASSIGN
      a
     :=
      5
    END STMT ASSIGN
   END STMT BLOCK
  END PROCEDURE a
 END GLOBAL SYM TABLE VARS
 BEGIN GLOBAL SYM TABLE TYPES
  TYPE boolean
  TYPE integer
  TYPE float
 END GLOBAL SYM TABLE TYPES
END GLOBAL SYM TABLE
